### DISCUSSION TITLE
add email to list

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [community/community#125071] [GitHub Education]
### Hi everyone,

how do  i add my school email to the list or request it to be?

**Comment by davevad93** [comment]
Hi @kittenvr 

Go to `Settings` => `Emails` => `Add email address`

![ll](https://github.com/community/community/assets/94308234/5d332aa6-3a14-4bd9-9b64-a58059ed49c1)

**Reply by pizzav-xyz** [reply]
i mean to the list

**Reply by davevad93** [reply]
Ah sorry @kittenvr , I didn't understand what you meant, if you're talking about about the edu email to get your students benefits you have to set it as a primary email in the settings and then it will be automatically selected on the application page. That's what I did when I made my request to GitHub Education.

**Reply by pizzav-xyz** [reply]
no like add it to the list off verified emails for github becuase i think it is not in there like intellij ultimate swot

**Reply by davevad93** [reply]
Hey @kittenvr , but you're talking about verifying your school email address in GitHub, right? If so take a look at [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/verifying-your-email-address).  
If that's not what you're referring to I'm really sorry but I'm not getting it..

**Reply by pizzav-xyz** [reply]
https://github.com/JetBrains/swot

**Reply by davevad93** [reply]
Ok. so if i'm correct what you're talking about is to add your educational institution domain to the [swot](https://github.com/JetBrains/swot) repo list, in order to do that correctly follow these steps:

1. `Fork` the repo
2. `Clone` the repo to your local environment
3. This step is important so pay close attention.
   1. As stated in the `README` '***the repository reverses the order of the name components Example: add file /lib/domains/ng/edu/unaab.txt to add a domain unaab.edu.ng to the repository***.' So for example if the domain of your school is  `school.eu` you should add in `/lib/domains/` a folder named `eu` with a `txt` file named `school`, if the domain of your school is  `edu.school.eu` you should add in `/lib/domains/` a folder named `eu` with inside another folder named `school` with inside a `txt` file named `edu`.
   2. As stated in the `README` '***Each file in the repository has to contain the name of the university, which the domain represented by the file belongs to. For example, a file /lib/domains/ng/edu/unaab.txt has to contain a line in it: "Federal University of Agriculture, Abeokuta"***' so that basically means that your `txt` file should include a line with the name of your educational institution.
4. After you've added the necessary folder(s) and the `txt` file **without altering the structure of the repo** you can push the changes  back to your remote repo.
   1. `git add .`
   2. `git commit -m "some message"`
   3. `git push origin master`
6. Make a `pull request` with the changes.

Hope this helps.

**Reply by pizzav-xyz** [reply]
you are not i want to know if there is a varient of this for github education

**Reply by davevad93** [reply]
Ah ok @kittenvr, I don't think so but I'm not 100% sure, maybe someone else here in the community might know more about this, sorry!

**Comment by Janethq** [comment]
If you want to request your school email to be added to GitHub Education, you should follow the process outlined in the GitHub Education documentation:

1. Go to [GitHub Education](https://education.github.com/).
2. Click on "Benefits" in the top navigation bar.
3. Under "Individuals", click "Get teacher benefits".
4. Select or add the email address you use for school.
5. Click "Continue", and then you will be prompted to upload proof of your academic status.
6. Enter your school's name.
7. Describe how you plan to use GitHub.
8. Verify your application details, then click "Process my application".

If your application is approved, your school email address will be added to GitHub Education.

**Comment by winneroyal** [comment]
Humanity could only watch in paralyzed horror as the [Hellstar](https://hellstar-clothing.com/) filled the entire sky, casting Earth into an unending blood-red twilight, its surface teeming with vast, writhing tendrils that reached down like grotesque fingers, pulling cities, mountains, and seas into its ravenous maw.
